src/routers/handlers/individual-psc-list/individualPscListHandler.ts

```diff
@@ -84,13 +83,15 @@ export class IndividualPscListHandler extends GenericHandler<IndividualPscListVi
         logger.info(`${IndividualPscListHandler.name} - ${this.executePost.name} called for transaction: ${req.params?.transactionId} and submissionId: ${req.params?.submissionId}`);
 
         const pscSelected = req.body.pscSelect;
+        logger.debug(`${IndividualPscListHandler.name} - ${this.executePost.name} - patching submission resource for transaction: ${req.params.transactionId} and submissionId: ${req.params.submissionId} with PSC ID: ${pscSelected}`);
```

Comment:
The` if` clause was originally added to prevent the user from proceeding without selecting a PSC, but this validation can be picked up during cloudy day scenarios as we are still able to navigate back to the PSC list page.